cmd/gaia/testnets/README.md

```diff
@@ -165,64 +109,64 @@ Check the everything is running smoothly:
 gaiacli status
 ```
 
-## Generate keys
+View the status of the network with the [Cosmos Explorer](https://explorecosmos.network).Once your full node syncs up to the current block height, you should see it appear on the [list of full nodes](https://explorecosmos.network/validators).
+
+## Generate Keys
 
 You'll need a private and public key pair \(a.k.a. `sk, pk` respectively\) to be able to receive funds, send txs, bond tx, etc.
 
-To generate your a new key \(default _ed25519_ elliptic curve\):
+To generate a new key \(default _ed25519_ elliptic curve\):
 
 ```
 gaiacli keys add <your_key_name>
 ```
 
-Next, you will have to enter a passphrase for your key twice. Save the _seed_ _phrase_ in a safe place in case you forget the password.
+Next, you will have to create a passphrase. Save the _seed_ _phrase_ in a safe place in case you forget the password.
 
-Now if you check your private keys you will see the `<your_key_name>` key among them:
+Now if you check your private keys you will see `<your_key_name>` among them:
 
 ```
 gaiacli keys show <your_key_name>
 ```
 
-You can see all your other available keys by typing:
+You can see all your available keys by typing:
 
 ```
 gaiacli keys list
 ```
 
-The validator pubkey from your node should be the same as the one printed with the command:
+The validator pubkey for your node is:
 
 ```
 gaiad tendermint show_validator
 ```
 
-Finally, save your address and pubkey into a variable to use them afterwards.
+Lastly, save your address and pubkey to environment variables for later use.
 
 ```
 MYADDR=<your_newly_generated_address>
 MYPUBKEY=<your_newly_generated_public_key>
 ```
 
-**IMPORTANT:** We strongly recommend to **NOT** use the same passphrase for your different keys. The Tendermint team and the Interchain Foundation will not be responsible for the lost of funds.
+**IMPORTANT:** We strongly recommend **NOT** using the same passphrase for all your keys. The Tendermint team and the Interchain Foundation will not be responsible for the loss of funds.
 
-### Get coins
+### Get Tokens
 
-The best way to get coins at the moment is to ask in Riot chat. We plan to have a reliable faucet in future testnets.
+The best way to get tokens is from the [Cosmos Testnet Faucet](https://faucetcosmos.network). If the faucet is not working for you, try asking [#cosmos-validators](https://riot.im/app/#/room/#cosmos-validators:matrix.org).
 
-## Send tokens
+## Send Tokens
 
 ```
-gaiacli send --amount=1000fermion --chain-id=<name_of_testnet_chain> --sequence=0 --name=<key_name> --to=<destination_address>
+gaiacli send --amount=10faucetToken --chain-id=<name_of_testnet_chain> --name=<key_name> --to=<destination_address>
 ```
 
-The `--amount` flag defines the corresponding amount of the coin in the format `--amount=<value|coin_name>`
+Note: The `--amount` flag accepts the format `--amount=<value|coin_name>`.
 
-The `--sequence` flag corresponds to the sequence number to sign the tx.
```

Comment:
I think it is still valuable to show that sequence numbers are a thing. Just add a line afterwards saying that if the sequence flag is omitted, it will auto-increment.

Reply:
From a usability standpoint, this readme is not documentation for `gaiacli`. My goal with these changes is to simplify onboarding for full nodes and delegators. From what I can tell, the `--sequence` is no longer required for any command, so it's best to omit it. That's one less feature users have to worry about so they can focus on other things.

We should have comprehensive `gaiacli` and `gaiad` docs in another file, in my opinion.